mirrord/protocol/src/tcp.rs

```diff
@@ -272,6 +395,11 @@ impl HttpRequestFallback {
 pub static HTTP_FRAMED_VERSION: LazyLock<VersionReq> =
     LazyLock::new(|| ">=1.3.0".parse().expect("Bad Identifier"));
 
+/// Minimal mirrord-protocol version that allows [`DaemonTcp::HttpRequestChunked`] instead of
+/// [`DaemonTcp::HttpRequest`].
+pub static HTTP_CHUNKED_VERSION: LazyLock<VersionReq> =
+    LazyLock::new(|| ">=1.6.0".parse().expect("Bad Identifier")); // TODO: check version number
```

Comment:
Delett' the TODO
Also, I think Alex was quicker and already bumped the crate to `1.6.0`. Gotta bump again